cfme/containers/provider/openshift.py

```diff
@@ -137,6 +158,23 @@ def __init__(
             endpoints=endpoints,
             appliance=appliance)
 
+    def create(self, **kwargs):
+
+        # Enable alerts collection before adding the provider to avoid missing active
+        # alert after adding the provider
+        # For more info: https://bugzilla.redhat.com/show_bug.cgi?id=1514950
+        if self.appliance.version >= '5.9' and getattr(self, "alerts_type") == "Prometheus":
```

Comment:
**Question**

Is this required when alerts_type OR metrics_type are prometheus? Or do these alert profiles only need to be set with Prometheus alerts, but not with Prometheus metrics?

Reply:
Assigning alert profiles is required only for alerts, 

By assigning the profiles you allow MIQ to pay attention to Prometheus alerts, If an alert endpoint is added without assigning both profiles, MIQ will collect the alerts from Prometheus and ignore them.